tests/suites/test_suite_pk.function

```diff
@@ -710,36 +710,76 @@ exit:
 /* END_CASE */
 
 /* BEGIN_CASE depends_on:MBEDTLS_RSA_C */
-void pk_rsa_encrypt_test_vec( data_t * message, int mod, int radix_N,
-                              char * input_N, int radix_E, char * input_E,
-                              data_t * result, int ret )
+void pk_rsa_encrypt_decrypt_test( data_t * message, int mod, int radix_P,
+                                  char * input_P, int radix_Q, char * input_Q,
+                                  int radix_N, char * input_N, int radix_E,
+                                  char * input_E, int ret )
 {
-    unsigned char output[300];
+    unsigned char output[300], result[300];
     mbedtls_test_rnd_pseudo_info rnd_info;
+    mbedtls_mpi N, P, Q, E;
     mbedtls_rsa_context *rsa;
     mbedtls_pk_context pk;
-    size_t olen;
+    size_t olen, rlen;
+
+    mbedtls_pk_init( &pk );
 
     memset( &rnd_info,  0, sizeof( mbedtls_test_rnd_pseudo_info ) );
     memset( output,     0, sizeof( output ) );
 
+    USE_PSA_INIT( );
+
+    /* encryption test */
 
-    mbedtls_pk_init( &pk );
+    /* init pk-rsa context */
     TEST_ASSERT( mbedtls_pk_setup( &pk, mbedtls_pk_info_from_type( MBEDTLS_PK_RSA ) ) == 0 );
     rsa = mbedtls_pk_rsa( pk );
 
+    /* load public key */
     rsa->len = mod / 8;
     TEST_ASSERT( mbedtls_test_read_mpi( &rsa->N, radix_N, input_N ) == 0 );
     TEST_ASSERT( mbedtls_test_read_mpi( &rsa->E, radix_E, input_E ) == 0 );
 
     TEST_ASSERT( mbedtls_pk_encrypt( &pk, message->x, message->len,
                             output, &olen, sizeof( output ),
                             mbedtls_test_rnd_pseudo_rand, &rnd_info ) == ret );
-    TEST_ASSERT( olen == result->len );
-    TEST_ASSERT( memcmp( output, result->x, olen ) == 0 );
+
+    /* decryption test */
+    mbedtls_mpi_init( &N ); mbedtls_mpi_init( &P );
+    mbedtls_mpi_init( &Q ); mbedtls_mpi_init( &E );
+
+    /* init pk-rsa context */
+    mbedtls_pk_free( &pk );
+    TEST_ASSERT( mbedtls_pk_setup( &pk, mbedtls_pk_info_from_type( MBEDTLS_PK_RSA ) ) == 0 );
```

Comment:
Line exceeds 80 char limit.

Reply:
Seems that this comment has not been addressed.

Reply:
I do see it in the changes: https://github.com/ARMmbed/mbedtls/pull/5525/files#diff-ed44a66973ca9c92032696551afcbba5d2e157df5fa239b64edd1de4c988f488R751

Reply:
I think you mean https://github.com/ARMmbed/mbedtls/pull/5525/files#diff-ed44a66973ca9c92032696551afcbba5d2e157df5fa239b64edd1de4c988f488R734 I can also change it

Reply:
I checked that `TEST_ASSERT( mbedtls_pk_setup( &pk, mbedtls_pk_info_from_type( MBEDTLS_PK_RSA ) ) == 0 );` still exists:
https://github.com/ARMmbed/mbedtls/pull/5525/files#diff-ed44a66973ca9c92032696551afcbba5d2e157df5fa239b64edd1de4c988f488R735
But this is existing code.

Reply:
Yes, I think we're sometimes less strict about style (including line length) in test code than in library code. We should probably be more strict for new code, but fixing pre-existing code is probably not worth it unless it comes naturally (like, you're touching that line anyway).